var/spack/repos/builtin/packages/py-wsproto/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyWsproto(PythonPackage):
+    """This repository contains a pure-Python implementation of
+    a WebSocket protocol stack. It’s written from the ground up
```

Comment:
```suggestion
    a WebSocket protocol stack. It's written from the ground up
```
This unicode char is causing the Python 2.7 tests to fail